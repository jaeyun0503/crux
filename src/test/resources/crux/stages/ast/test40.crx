// test comparisons

fn test(a: int, b: int) {
  printBool(a >= b);
  println();
  printBool(a <= b);
  println();
  printBool(a != b);
  println();
  printBool(a == b);
  println();
  printBool(a < b);
  println();
  printBool(a > b);
  println();
}

fn main() {
  let i: int;
  let j: int;
  i = 0;
  while (i <= 4) {
    j = 0;
    while (j <= 4) {
       test(i, j);
       j = j + 1;
    }
    i = i + 1;
  }
}
