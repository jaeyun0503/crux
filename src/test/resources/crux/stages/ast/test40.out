
test(a, b) {
  printBool((a >= b));
  println();
  printBool((a <= b));
  println();
  printBool((a != b));
  println();
  printBool((a == b));
  println();
  printBool((a < b));
  println();
  printBool((a > b));
  println();
}
main() {
  i;
  j;
  i = 0;
  while ((i <= 4)) {
    j = 0;
    while ((j <= 4)) {
      test(i, j);
      j = (j + 1);
    }
    i = (i + 1);
  }
}

